src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -128,333 +128,336 @@
 import org.bukkit.entity.ZombieHorse;
 import org.eclipse.jdt.annotation.Nullable;
 
-import ch.njol.skript.Skript;
-import ch.njol.skript.lang.Literal;
-import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.yggdrasil.Fields;
+import java.io.NotSerializableException;
+import java.io.StreamCorruptedException;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * @author Peter Güttinger
  */
 public class SimpleEntityData extends EntityData<Entity> {
```

Comment:
What has been changed in this class?

Reply:
This file contains tab changes, as the file originally contained tabs that were replaced by spaces. I think it would be useful to solve this problem to restore proper diffs and be able to review properly 😀